match2/wikis/rocketleague/match_summary.lua

```diff
@@ -114,6 +123,12 @@ function p.getByMatchId(args)
 	return wrapper
 end
 
+function p._gameHeader(header)
```

Comment:
Is it correct that this isn't currently used?

Reply:
good catch, forgot to remove it